physical/physical.go

```diff
@@ -175,6 +199,12 @@ func (le *LogicExpression) Materialize(ctx context.Context, matCtx *Materializat
 	return execution.NewLogicExpression(materialized), nil
 }
 
+func (le *LogicExpression) Visualize() *graph.Node {
+	n := graph.NewNode("Logic Expression")
+	n.AddChild("source", le.Formula.Visualize())
```

Comment:
in physical.Filter field of type Formula got "predicate" name, and here is "source"

Reply:
Indeed, I tried to use the name "source" a lot, to not confuse users.